src/Logging/LoggingHandler.cs

```diff
@@ -105,6 +105,9 @@ internal LoggingHandler(GoogleAdsConfig config)
                     RequestHeaders = context.Options.Headers,  // includes the RequestId
                     IsFailure = oldTask.IsFaulted,
                     Exception = exception,
+                    CustomerId = GetCustomerId(request),
+                    ResponseHeaders = Merge(GetResponseHeader(call.ResponseHeadersAsync),
+                                            call.GetTrailers()),
```

Comment:
For AsyncUnary, this is fine, because there's only one response body to be read, so you'd be at the end of response stream by the time the GetTrailers() is called. This doesn't work with Streaming, but there's no way to check EOS status either, which is why we need a TryGetTrailers() instead.